src/CqlParser.ts

```diff
@@ -386,6 +388,8 @@ export class CqlParser {
           value = `'${value}'`;
         }
         return `${filter[1]} ${cqlOperator} ${value}`;
+      case '<=x<=':
+        return `${filter[1]} ${cqlOperator} ${filter[2]} AND ${filter[3]}`;
```

Comment:
CQL also knows the `BETWEEN` operator, perhaps it makes sense to use it here, too. See https://docs.geoserver.org/latest/en/user/styling/ysld/reference/filters.html

Reply:
That is already the BETWEEN operator. From above

```
const cqlOperator = operatorReverseMap[operator];
```

See also tests

Reply:
:man_facepalming: Yes, shouldn't review things early in the morning :rofl: